docs/GUIDE_FOR_KOGITO_DEVS.md

```diff
@@ -0,0 +1,352 @@
+# Introduction
+This is a list of tasks/activities that are useful or necessary 
+to know while working on the Kogito operator. 
+This document is written in an order that is recommended to 
+be worked through chronologically as a new Kogito operator developer.
+
+# Installing Kogito Dependencies
+## CRC
```

Comment:
I think it's worth mentioning KIND or minikube since some people might not have the resources to install CRC.

Reply:
What do you think of [K3s](https://k3s.io/)? It has an even lower system requirement and faster startup [compared to minikube/kind](https://brennerm.github.io/posts/minikube-vs-kind-vs-k3s.html#summary).

Reply:
What do you think of [K3s](https://k3s.io/)? It has an even lower system requirement and faster startup [compared to minikube/kind](https://brennerm.github.io/posts/minikube-vs-kind-vs-k3s.html#summary).

Reply:
Based on feedback from the team and personal experience, I added minikube as an alternative in the guide since it can also be deployed as a container for the faster startup time, and it just seems more polished in general.